x-pack/plugins/triggers_actions_ui/public/application/sections/rule_details/components/rule_details.tsx

```diff
@@ -340,175 +337,63 @@ export const RuleDetails: React.FunctionComponent<RuleDetailsProps> = ({
+              disableRule={async () => await disableRule(rule)}
+              enableRule={async () => await enableRule(rule)}
+              snoozeRule={async (snoozeEndTime: string | -1) =>
+                await snoozeRule(rule, snoozeEndTime)
+              }
+              unsnoozeRule={async () => await unsnoozeRule(rule)}
+              item={rule as RuleTableItem}
+              onRuleChanged={requestRefresh}
+              direction="row"
+              isEditable={hasEditButton}
+              previousSnoozeInterval={null}
+            />
           </EuiFlexItem>
         </EuiFlexGroup>
-        {rule.enabled && !dismissRuleErrors && rule.executionStatus.status === 'error' ? (
+        {rule.enabled &&
+        rule.executionStatus.error?.reason === AlertExecutionStatusErrorReasons.License ? (
           <EuiFlexGroup>
             <EuiFlexItem>
-              <EuiCallOut
-                color="danger"
-                data-test-subj="ruleErrorBanner"
-                size="s"
-                title={getRuleStatusErrorReasonText()}
-                iconType="rule"
-              >
-                <EuiText size="s" color="danger" data-test-subj="ruleErrorMessageText">
-                  {rule.executionStatus.error?.message}
-                </EuiText>
-                <EuiSpacer size="s" />
-                <EuiFlexGroup gutterSize="s" wrap={true}>
-                  <EuiFlexItem grow={false}>
-                    <EuiButton
-                      data-test-subj="dismiss-execution-error"
-                      color="danger"
-                      onClick={() => setDismissRuleErrors(true)}
-                    >
-                      <FormattedMessage
-                        id="xpack.triggersActionsUI.sections.ruleDetails.dismissButtonTitle"
-                        defaultMessage="Dismiss"
-                      />
-                    </EuiButton>
-                  </EuiFlexItem>
-                  {rule.executionStatus.error?.reason ===
-                    AlertExecutionStatusErrorReasons.License && (
-                    <EuiFlexItem grow={false}>
-                      <EuiButtonEmpty
-                        href={`${http.basePath.get()}/app/management/stack/license_management`}
-                        color="danger"
-                        target="_blank"
-                      >
-                        <FormattedMessage
-                          id="xpack.triggersActionsUI.sections.ruleDetails.manageLicensePlanBannerLinkTitle"
-                          defaultMessage="Manage license"
-                        />
-                      </EuiButtonEmpty>
-                    </EuiFlexItem>
-                  )}
-                </EuiFlexGroup>
+              <EuiCallOut color="danger" data-test-subj="ruleErrorBanner" size="s" iconType="rule">
+                <p>
+                  <EuiIcon color="danger" type="alert" />
+                  &nbsp;
+                  {getRuleStatusErrorReasonText()}
```

Comment:
This used to also show the `rule.executionStatus.error?.message` which said something about which license is required.  Now it just says `Cannot run rule` with no mention of license except for the link to the license page

Should we say something more than this?

<img width="1586" alt="Screen Shot 2022-04-01 at 11 10 38 AM" src="https://user-images.githubusercontent.com/13104637/161291442-0c4afec9-c53c-4c14-9eb5-fbffc9cd184c.png">